install.sh

```diff
@@ -39,6 +39,29 @@ then
   abort 'Bash must not run in POSIX mode. Please unset POSIXLY_CORRECT and try again.'
 fi
 
+usage() {
+  cat <<EOS
+Homebrew Installer
+Usage: [NONINTERACTIVE=1] [CI=1] $0 [options]
+    -h, --help       Display this message.
+    NONINTERACTIVE   Install without prompting.
+    CI               Imply NONINTERACTIVE.
```

Comment:
```suggestion
    CI               Install in CI mode (e.g. do not prompt for user input)
```

Reply:
Is there a difference between CI and NONINTERACTIVE?

Reply:
Not currently but want to leave the door open for it in future. There's a semantic difference.

Reply:
What’s the semantic difference? To the best of my understanding, CI needs to be non-interactive but not sure what else might be into it.

Also, in `uninstall.sh` they make it clear that `NONINTERACTIVE` just implies `--force` so I adopted the same idea.

Reply:
> CI needs to be non-interactive but not sure what else might be into it.

CI does need to be non-interactive but we may additionally change behaviour in future.